cmd/docker/docker.go

```diff
@@ -57,6 +57,7 @@ func newDockerCommand(dockerCli *command.DockerCli) *cobra.Command {
 	flags = cmd.Flags()
 	flags.BoolVarP(&opts.Version, "version", "v", false, "Print version information and quit")
 	flags.StringVar(&opts.ConfigDir, "config", cliconfig.Dir(), "Location of client config files")
+	flags.BoolVarP(&opts.TLSGetPass, "tlsgetpass", "", cliconfig.TLSGetPass(), "Request passphrase if needed to decrypt TLS keys")
```

Comment:
Thank you for the review. Yes, the PR checks whether a private key is encrypted. 
I added the flag as the PR seems to make a behavior change, i.e. encountering an encrypted private key will result in the CLI prompting for a passphrase, and failing in such situations may be the desired behavior in some cases? (perhaps in a shell script?) 
Please advise whether you consider this a valid reason or if I should go ahead and remove the flag.